### DISCUSSION TITLE
Keyword not supported: 'host', PostgresSql

### DISCUSSION THREAD

**Discussion by monet820** [testcontainers/testcontainers-dotnet#846] [Q&A] [answered]
Testcontainers.PostgreSql

private readonly PostgreSqlContainer _postgreSqlContainer = new PostgreSqlBuilder()

_postgreSqlContainer.GetConnectionString

Returns Host, should this not be Server?

I get the error in title when using Respawn to reset my database tables for testing.

**Comment by HofmeisterAn** [comment, marked as answer]
Which data provider do you use? According to the Npgsql [documentation](https://www.npgsql.org/doc/connection-string-parameters.html) the parameter is `Host`.

**Comment by monet820** [comment]
I currently use Dapper, FluentMigrator, Respawn with DbAdapter.Postgres and the NpgsqlConnection from Npgsql
Hopefully that answers the question.

**Reply by HofmeisterAn** [reply]
Cannot say much about your the configuration, but `NpgsqlConnection` runs fine in our tests:

https://github.com/testcontainers/testcontainers-dotnet/blob/710987a2562f7bb5a58d64bb74cba17cc8681db4/tests/Testcontainers.PostgreSql.Tests/PostgreSqlContainerTest.cs#L17-L29

**Comment by monet820** [comment]
Yea, everything works fine here also, expect for Respawn..  
Might just need to debug further :) 

Sorry to bother you.

**Comment by monet820** [comment]
I figured it out, it was Respawn.
Apparently i were creating respawn before I had created the tables.

To anyone else:
Moved Respawner.CreateAsync to my dispose method and its now working 👍
